communications/newsletters/newsletter_12_Nov2019.md

```diff
@@ -0,0 +1,100 @@
+
+Find details on these topics and more news from the community below 👇
+
+## Community meetings
+
+### Collaboration Cafes
+
+The next Collaboration Cafes will be held on 20 November 2019 at 19:00  [your local time](https://arewemeetingyet.com/London/2019-11-20/19:00/TuringWay-CollaborationCafe).
+Two more cafes have been scheduled for this year on 4 December 2019 at 15:00  [your local time](https://arewemeetingyet.com/London/2019-12-04/15:00/TuringWay-CollaborationCafe), and 18 December 2019 at 19:00 [your local time](https://arewemeetingyet.com/London/2019-11-20/19:00/TuringWay-CollaborationCafe).
+Please save these dates and sign up for the next Collaboration Cafe on [this HackMD](https://hackmd.io/@KirstieJane/CollabCafe).
+
+
+### Save the date: Turing Way Book Dash in February 2019
+
+The next Book Dash will take place on `[<Date>]` at `[<Location>]`.
+A Book Dash is an in-person collaborative event that brings together participants who are enthusiastic about data science and reproducibility and want to contribute to the improvement of *The Turing Way* book.
+We will share more information regarding the applications in December, but meanwhile, please check the reports from the [Manchester book dash](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/book-dash-mcr-report.md) and [London book dash](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/book-dash-ldn-report.md), summarising the participants' contributions during the event and their feedback.
+
+## News from the community
+
+### Turing Way is Expanding!
+
+We are excited to announce that *The Turing Way* is expanding into [a series of books](https://github.com/alan-turing-institute/the-turing-way).
+In addition to its previous focus on reproducibility, this project will aim to capture the vast amounts of knowledge of data science practices in the areas of project design, communication, outreach and ethics.
+Following an open source, community driven approach, we will welcome contributions from anyone from within the Turing community and beyond  who can support researchers and practitioners in learning or improving the "how" of conducting world-leading research in data science.
+The project proposal with further details is available [online](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/tps-funding-application-20190429.md).
+
+To plan the next steps, the core team met last month on 11 and 12 October to brainstorm and discuss the next steps in this project.
+The participants of this meeting explored a number of topics in four themed sessions, draft report from which is available on [this HackMD](https://hackmd.io/zVTeKhG2SIiBLam1YtILNg).
```

Comment:
Maybe make the link read only?

Reply:
And remove link to other hackmd?

Reply:
Good point. I will do that. I have asked @LouiseABowler to review the report when she has the time.

Reply:
The report is looking good @malvikasharan! 🚀 I've left one comment, but that's the only thing I can remember having a discussion about that wasn't already captured  - you've done a great job getting everything into the doc 😄 

I can't edit the file any more so left a couple of typos unfixed, but they're very minor for that amount of content.

Reply:
Thank you @LouiseABowler. My apology for closing the edits without realising that I requested you to edit 🤦‍♂

Reply:
Really awesome work - thank you both of you!!